lib/addons/topics-api.ts

```diff
@@ -0,0 +1,40 @@
+import OptableSDK from "../sdk";
+
+declare module "../sdk" {
+    export interface OptableSDK {
+        tryTopicsAPI: () => void;
+    }
+}
+
+declare global {
+    interface Document {
+        browsingTopics: () => Promise<Array<{
+            configVersion: string;
+            modelVersion: string;
+            taxonomyVersion: string;
+            topic: number;
+            version: string;
+        }>>;
+        featurePolicy:{ allowsFeature: (feature:string) => boolean };
+    }
+}
+
+OptableSDK.prototype.tryTopicsAPI = async function () {
+    if (!sessionStorage.topics_fetched && 'browsingTopics' in document && document.featurePolicy.allowsFeature('browsing-topics')) {
```

Comment:
I never used sessionStorage much, but it looks like the way to use it is through `setItem` / `getItem` ? If we are to keep that flag would it be better to avoid using global vars and instead store on `this`?

Reply:
`this` won't be persistent across pageviews
sessionStorage is the same as localStorage, but is wiped as soon as the user leaves the site/closes the browser/tab.
`setItem`/`getItem` is superfluous.